### ISSUE TITLE
Swift Compiler Error in XCode: Cannot find 'AnalyticsEventScreenView' in scope

### ISSUE THREAD

**Issue by losciur** [capacitor-community/firebase-analytics#98]
Hi guys,
I have an ionic capacitor 3 app, and when I compile with Xcode 12.5 I got these errors:
```
Cannot find 'AnalyticsEventScreenView' in scope
Cannot find 'AnalyticsParameterScreenName' in scope
Cannot find 'AnalyticsParameterScreenClass' in scope
```

Here a screenshot:

<img width="1491" alt="Schermata 2021-06-08 alle 11 53 12" src="https://user-images.githubusercontent.com/62714342/121165045-44138180-c850-11eb-96e7-4d8badd7d238.png">

Thi is my `ionic info`:

```
Ionic:

   Ionic CLI                     : 6.16.2 (/Users/stefmase/.nvm/versions/node/v14.16.1/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.6.8
   @angular-devkit/build-angular : 12.0.3
   @angular-devkit/schematics    : 12.0.3
   @angular/cli                  : 12.0.3
   @ionic/angular-toolkit        : 4.0.0

Capacitor:

   Capacitor CLI      : 3.0.0
   @capacitor/android : 3.0.0
   @capacitor/core    : 3.0.0
   @capacitor/ios     : 3.0.0

Utility:

   cordova-res                          : 0.15.3
   native-run (update available: 1.3.1) : 1.3.0

System:

   NodeJS : v14.16.1 (/Users/stefmase/.nvm/versions/node/v14.16.1/bin/node)
   npm    : 6.14.12
   OS     : macOS Big Sur

```

Thanks for the help.

**Comment by brownoxford** [comment]
Can you please try `@capacitor-community/firebase-analytics@1.0.1-alpha.0`? I've locked the pod dependency to the latest v8 of upstream Firebase/Analytics.

If you get this error compiling, you may need to run `pod repo update` to ensure your source repos are up to date:

```CocoaPods could not find compatible versions for pod "Firebase/Analytics"```

**Comment by losciur** [comment]
Great!!
it works!

I first got an error in compiling with ionic, then I run `pod repo update`and now it is all ok and also Xcode run the app without problems.

Will you merge @capacitor-community/firebase-analytics@1.0.1-alpha.0 into @capacitor-community/firebase-analytics@latest?

Thanks!

**Comment by brownoxford** [comment]
Thanks for testing @losciur , I'm working on another issue and will probably merge both when done - hopefully this week.

@all-contributors please add @losciur for bug, test

**Comment by losciur** [comment]
Thank you guys!!!
:smile:

**Comment by mvignesInflexsys** [comment]
> Can you please try `@capacitor-community/firebase-analytics@1.0.1-alpha.0`? I've locked the pod dependency to the latest v8 of upstream Firebase/Analytics.
> 
> If you get this error compiling, you may need to run `pod repo update` to ensure your source repos are up to date:
> 
> `CocoaPods could not find compatible versions for pod "Firebase/Analytics"`

Hi
I do that when I saw errors in Xcode but after `pod repo update`, got the same error on `npx cap sync`.
What can I do for it ?

```
[error] Analyzing dependencies
        Pre-downloading: `PRIVATE_PLUGIN` from
        `PRIVATE_REPO`,
        tag `7.0.0`
        [!] CocoaPods could not find compatible versions for pod
        "Firebase/Analytics":
        In Podfile:
        CapacitorCommunityFirebaseAnalytics (from
        `../../node_modules/@capacitor-community/firebase-analytics`) was
        resolved to 1.0.1-alpha.0, which depends on
        Firebase/Analytics (~> 8.0)
        
        CordovaPluginsStatic (from `../capacitor-cordova-ios-plugins`) was
        resolved to 3.0.2, which depends on
        Firebase/Analytics (~> 6.3.0)
```

Censored enterprise infos.
Thx

**Comment by mvignesInflexsys** [comment]
Got it.
Cause I'm passing a project from Cordova to Capacitor, it was a conflicted between @capacitor-community/firebase-analytics and cordova-plugin-firebase-analytics.

👍

**Comment by f1ght4fun** [comment]
@mvignesInflexsys how did you fix it!? I have same issue

**Comment by gbkwiatt** [comment]
I am having similar issue
```
In Podfile:
        CapacitorCommunityFirebaseAnalytics (from `../../node_modules/@capacitor-community/firebase-analytics`) was
        resolved to 1.0.1, which depends on
        Firebase/Analytics (~> 8.0)
        
        None of your spec sources contain a spec satisfying the dependency: `Firebase/Analytics (~> 8.0)`.
  ```

but if in `CapacitorCommunityFirebaseAnalytics` I remove `, '~> 8.0'` from `s.dependency 'Firebase/Analytics', '~> 8.0'`

It works fine, why is that ?

**Comment by jcesarmobile** [comment]
Closing since I can't reproduce the OPs problem and by the comments looks like it was fixed.
For the conflicts with Cordova plugins, I've released version 4.0.0 without a specific firebase dependency that might have fixed that problem, but can still cause build failures if the Cordova plugins use very old firebase dependencies.
